cmd/derper/derper.go

```diff
@@ -141,6 +152,14 @@ func writeNewConfig() config {
 	return cfg
 }
 
+func checkMeshKey(key string) (string, error) {
+	key = strings.TrimSpace(key)
+	if !validMeshKey.MatchString(key) {
+		return "", fmt.Errorf("key in %q must contain 64+ hex digits", key)
```

Comment:
We should not log the key. The previous regex validation only logged the filename it was contained in.

Reply:
Whoops, removing!

Reply:
Sorry, I missed that too.

Reply:
https://github.com/tailscale/tailscale/pull/14915